README.md

```diff
@@ -1,12 +1,14 @@
-## Algorithms Exercices List
+## Algorithms Exercise List
 
-We are developing Algorithms and Data Structures exercises to help people around the world, to improve logic skills. In this repository you will be able to create and test your algorithms. Look for all levels exercicies:
+We are developing Algorithms and Data Structures' exercises to help people around the world improve their logic skills. In this repository you will be able to create and test your algorithms. The levels of exercises are as follows:
 
  - Easy Level;
  - Medium Level;
  - Hard Level;
  - Refactoring Level.
 
+Look for all levels of exercises:
+
 To contribute to this project, create a PR. If tested and approved, your code will be inserted!
 
 *This project is created for Hacktober Fest 2020. [Read more](https://hacktoberfest.digitalocean.com/)*
```

Comment:
What do you think to update the Hacktober Fest year, and also the event logo? 😄

Reply:
sure, I apologize for failing to notice the year and logo. I will correct it as well. I will also try to add an exercise as well with a different PR. I'll leave this one for the documentation.